<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindCollateralTxDlg</class>
 <widget class="QDialog" name="FindCollateralTxDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#FindCollateralTxDlg{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 #FFFFFF,stop:1 #EEEEEE);
}
 
QRadioButton::indicator { 
    width: 12px;
    height: 12px
}
 
QRadioButton::indicator:checked { 
	image: url(:/images/radiobox_checked.png);
}
 
QRadioButton::indicator:unchecked {
	image: url(:/images/radiobox_normal.png);
}
 
QRadioButton::indicator:unchecked:hover { 
	image: url(:/images/radiobox_hot.png);
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QLabel" name="lblMessage">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Transaction ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="rdbFixed">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdbUnused">
       <property name="text">
        <string>可用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdbUsed">
       <property name="text">
        <string>所有</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSubmit">
       <property name="text">
        <string> 确 定 </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancle">
       <property name="text">
        <string> 取 消 </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
